kale/loaddata/video_access.py

```diff
@@ -169,16 +220,29 @@ class VideoDatasetAccess(DatasetAccess):
         frames_per_segment (int): length of each action sample (the unit is number of frame)
         n_classes (int): number of class
         transform_kind (string): types of video transforms
+        seed: (int): seed value set manually.
     """
 
-    def __init__(self, data_path, train_list, test_list, image_modality, frames_per_segment, n_classes, transform_kind):
+    def __init__(
+        self, data_path, train_list, test_list, image_modality, frames_per_segment, n_classes, transform_kind, seed
+    ):
         super().__init__(n_classes)
         self._data_path = data_path
         self._train_list = train_list
         self._test_list = test_list
         self._image_modality = image_modality
         self._frames_per_segment = frames_per_segment
         self._transform = video_transform.get_transform(transform_kind, self._image_modality)
+        self._seed = seed
+
+    def get_train_val(self, val_ratio):
+        """Get the train and validation dataset with the fixed random split."""
+        train_dataset = self.get_train()
+        ntotal = len(train_dataset)
+        ntrain = int((1 - val_ratio) * ntotal)
+        return torch.utils.data.random_split(
+            train_dataset, [ntrain, ntotal - ntrain], generator=torch.Generator().manual_seed(self._seed)
```

Comment:
seed has already been set earlier to no need to reset this, right? That's why I removed the manual_seed in dataset_access, its parent
So is this function needed?

Reply:
Yes, it is needed. We need to call the `get_train_val` twice for joint inputs and, therefore, need to keep these two random seeds the same.

Reply:
I see. This is tricky and good to know. Putting a comment explaining this may help future users.